transactions/update-contract/2023/may-3-rewards/README.md

```diff
@@ -0,0 +1,38 @@
+# Update FlowIDTableStaking Contract
```

Comment:
Could we document where the `bonusTokenAmount` argument value came from?

Reply:
Yep, I added that to the README